rocm_docs/SYNC_UPSTREAM.md

```diff
@@ -159,3 +162,13 @@ git push origin upstream
 ```
 
 Upon reaching here, the process is now complete.
+
+# Tips for debugging
+
+Bisecting between two weekly syncs is often necessary for debugging. The process should look like this:
+* Check out `merge-YYMMDD-prev` tag
+* Select a commit in the middle of the sync (between `merge-YYMMDD-prev` and `merge-YYMMDD`)
+* Merge to that commit point (i.e. perform a "mini sync")
+* Resolve merge conflicts based on what was done in that weekly sync
+* Test
+* Repeat until the required commit is found
```

Comment:
You can also provide reference to https://confluence.amd.com/display/~cchen104/XLA+hotfix+track to cherry-pick XLA hot fixes if the build breaks in xla